flask_project/campaign_manager/models/models.py

```diff
@@ -0,0 +1,298 @@
+                                     Base.metadata,
+                                     Column('campaign_id', Integer,
+                                            ForeignKey('campaign.id_campaign')
+                                            ),
+                                     Column('function_id', Integer,
+                                            ForeignKey('function.id_function')
+                                            ))
+
+
+class User(Base):
+
+    __tablename__ = 'user'
+
+    id = Column('id_user', Integer, primary_key=True,
+                autoincrement=True)
+    osm_user_id = Column(String(20), unique=True, nullable=False)
+    email = Column(String(40))
+    campaign_creator = relationship('Campaign', back_populates='creator',
+                                    lazy=True, uselist=False)
+    chat_sender = relationship('Chat', foreign_keys='Chat.sender_id',
+                               back_populates='sender', lazy=True,
+                               uselist=False)
+    chat_receiver = relationship('Chat', foreign_keys='Chat.receiver_id',
+                                 back_populates='reciever',
+                                 lazy=True, uselist=False)
+    notification_sender = relationship('Notification',
+                                       back_populates='sender',
+                                       lazy=True, uselist=False)
+    campaigns = relationship('Campaign', secondary=adminAssociations,
+                             lazy='subquery', back_populates='users')
+    teams = relationship('Team', secondary=teamUserAssociations,
+                         lazy='subquery', back_populates='users')
+
+    def __init__(self, **kwargs):
+        super(User, self).__init__(**kwargs)
+
+
+class Campaign(Base):
+
+    __tablename__ = 'campaign'
+
+    id = Column('id_campaign', Integer, primary_key=True,
+                autoincrement=True)
+    creator_id = Column(Integer, ForeignKey('user.id_user'),
+                        nullable=False)
+    creator = relationship('User', back_populates='campaign_creator')
+    users = relationship('User', secondary=adminAssociations,
+                         lazy='subquery', back_populates='campaigns')
+    featureTypes = relationship('FeatureType',
+                                secondary=typeCampaignAssociations,
+                                lazy='subquery',
+                                back_populates='campaigns')
+    functions = relationship('Function',
+                             secondary=functionCampaignAssociations,
+                             lazy='subquery', back_populates='campaigns'
+                             )
+    chat = relationship('Chat', back_populates='campaign', lazy=True)
+    notification = relationship('Notification',
+                                back_populates='campaign', lazy=True)
+    geometry = Column(Geometry('POLYGON'))
+    taskBoundaries = relationship('TaskBoundary',
```

Comment:
task_boundaries ?

Reply:
Thanks for pointing it out. I'll correct it asap.